tools/update-rc3.in

```diff
@@ -395,6 +395,9 @@ remove_image() {
     remove_checksum_cache '[^\:]*' '[^\:]*' "$1"
     remove_checksum_cache '[^\:]*' '[^\:]*' "$1"'\.[^\:]*'
     rm -f "$1"{,.md5,.sha,.sha1,.sha224,.sha256,.sha384,.sha512}{,-padded}{,.tmp} ${2:+"$2"}
+    # Also remove the image file manifest, if present
+    MANIFEST_FILE="`echo $1 | sed s/.squashfs/-manifest.json/`"
```

Comment:
Consider me olds-cool, but I prefer to see arguments to commands (double-)quoted, here and in other changes below. Helps avoid surprises (or worse) in the long run.

Reply:
fixed in latest commit